pages/Addons.md

```diff
@@ -73,6 +73,7 @@ Missile Warfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
 Supreme | RelativoBR | <details><summary>expand</summary>Supreme is an addon for Slimefun which adds 100+ various new resources that will allow you to craft powerful new items, weapons, tools and armor. These can be made up from titanium, aurum, adamantium, thornium with some being magical, rare, epic, legendary or supreme! It also adds 12 new electric generators, 5 new capacitors and even more... </details> | [Readme](https://github.com/RelativoBR/Supreme/blob/main/README.md) | [Github](https://github.com/RelativoBR/Supreme) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/Supreme/main/)
 Bump | SlimefunGuguProject | <details><summary>expand</summary>A Slimefun addon that adds magic weapons, food and an appraisal system. Through the appraisal system, you could even get an 8 star weapon or armor.</details> | [Readme](https://github.com/SlimefunGuguProject/Bump/blob/main/README.md) | [Github](https://github.com/SlimefunGuguProject/Bump) | [Builds (via Third-Party site)](https://builds.guizhanss.net/SlimefunGuguProject/Bump/main/)
+Netheopoiesis | Sefiraat / J3fftw | <details><summary>expand</summary>This addon was our entry into the Slimefun Addon Jam 2022. The premise of the addon is to allow players to play vanilla Minecraft and Slimefun solely within the Nether. You can purify and beautify the nether over time including changing biomes, spawns and lots lots more. This is held up by a robust plant growing/breeding and harvesting system with plants that fullfill many functions including active like spawnings, growing trees, dropping items and more. A branching but linear progression through the addon should provide lots of content, with more to come.</details> | [Readme](https://github.com/Sefiraat/Netheopoiesis/blob/master/README.md) | [Github](https://github.com/Sefiraat/Netheopoiesis) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Netheopoiesis/master/)
```

Comment:
I assume at least thats what it was supposed to be
also "spawnings" isn't a word afaik, not sure what you meant there though
```suggestion
Netheopoiesis | Sefiraat / J3fftw | <details><summary>expand</summary>This addon was our entry into the Slimefun Addon Jam 2022. The premise of the addon is to allow players to play vanilla Minecraft and Slimefun solely within the Nether. You can purify and beautify the nether over time including changing biomes, spawns and lots lots more. This is held up by a robust plant growing/breeding and harvesting system with plants that fullfill many functions including activities like spawnings, growing trees, dropping items and more. A branching but linear progression through the addon should provide lots of content, with more to come.</details> | [Readme](https://github.com/Sefiraat/Netheopoiesis/blob/master/README.md) | [Github](https://github.com/Sefiraat/Netheopoiesis) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Netheopoiesis/master/)
```

Reply:
How very dare you :D

Reply:
get englished